website/static/js/contribManager.js

```diff
@@ -113,6 +116,37 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
             id:self.id});
     };
 
+    self.addParentAdmin = function () {
+        // Immediately adds parent admin to the component with permissions=read and visible=True
+        var self = this;
```

Comment:
I don't think this declaration is necessary. 

`self = this` is set for the entire ViewModel [here](https://github.com/CenterForOpenScience/osf.io/pull/7831/files#diff-f4df2c5c528f874302fff1ee05a65e8aR42).

Reply:
ah, thanks!